.github/merge_rules.yaml

```diff
@@ -125,9 +125,6 @@
 
 - name: PrimTorch
   patterns:
-  - aten/src/ATen/native_functions.yaml
-  - aten/src/ATen/native/**
-  - test/**
```

Comment:
These are too general and block people's merges, as here: https://github.com/pytorch/pytorch/pull/110150#issuecomment-1760566745